lib/preprocessors/glimmer.js

```diff
@@ -22,11 +17,16 @@
 const oneLineTemplateRegex = /\[__GLIMMER_TEMPLATE\(`(.*)`, {.*}\)]/;
 
 // regex to capture opening template tag
-const openingTemplateTagRegex = /\[__GLIMMER_TEMPLATE\(`$/;
+const openingTemplateTagRegex = /\[__GLIMMER_TEMPLATE\(`/;
 
 // regex to capture closing template tag
 const closingTemplateTagRegex = /`, {.*}\)]/;
 
+// regex with capture group on scope token variables. In the following example:
+// `, { strictMode: true, scope: () => ({on,noop}) })]
+// the capture group would contain `on,noop`
+const getScopeTokenRegex = /scope: \(\) => \({([\d,A-z]*)}\)/;
```

Comment:
Should we mark this as a false positive, or can the regexp be improved?

Reply:
@bmish  looked into it this morning, the issue with github validation is that `A-z` also contains the characters _, `, [, ], and ^.

This can be simplified to just use:
```
/scope: \(\) => \({([\w,]*)}\)/;
```
which will capture _everything_ inside the scope function. the original attempt tried to match only valid scope characters, but I actually don't think that will be necessary